src/main/java/com/jfb/tickets/config/TicketsContextConfiguration.java

```diff
@@ -0,0 +1,39 @@
+package com.jfb.tickets.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+import com.jfb.tickets.dao.TicketDAO;
+import com.jfb.tickets.dao.UserDAO;
+import com.jfb.tickets.service.TicketService;
+import com.jfb.tickets.service.UserService;
+
+@Configuration
+public class TicketsContextConfiguration {
+    @Bean
+    @Scope("singleton")
+    public DatabaseConnection dataSource() {
+        return new DatabaseConnection();
```

Comment:
instead of manual configuration of Connection and choosing Driver manager use DataSource specific db bean:
```java
    @Bean
    public DataSource dataSource(){
        PGSimpleDataSource dataSource = new PGSimpleDataSource();
        dataSource.setURL(...);
        dataSource.setUser(...);
        dataSource.setPassword(...);
        return dataSource;
    }
```